### PR TITLE
sound150@claudiux: Rebase and update for cinnamon 5.4 (add tooltip section)

### LINKED ISSUE #4611: sound150@claudiux not showing album art in mint 21.1
```
 * Applet Version 2.0.0 (2022-06-07 07:26:49)
 * Cinnamon 5.6.5
 * Distribution - Linux Mint 21.1 x86_64 
 * Snippet or attachment of ~/.cinnamon/glass.log  does not exist
 * Snippet or attachment of ~/.xsession-errors  NONE

 ```

**Notify author of applet**
@claudiux 

**Issue**
Mint 21.1 sound 150 applet does not show mp3 album art.

**Steps to reproduce**
Click on an mp3 with album art and play.   Click on the sound icon in the panel.  No album art appears, only title/artist and controls etc.

**Expected behaviour**
Album art should appear behind title overlay

**Other information**
The normal sound applet does work, just sound 150+ that has the issue.
I tried multiple themes just in case.
sound150

### PR DESCRIPTION
**Pull request by rcalixte**
Also updated the translation template for 5.4 (will need assistance to update the other languages)
@claudiux

Fixes #4611

(Rebasing and reopening #4501 since I closed it when I thought it wasn't necessary but it looks like it would help. I find the default Sound applet has the needed functionality.)

### CODE REVIEW

**Comment by claudiux** [comment]
Hi @rcalixte 

Thank you for the care!

I'll update the fr.po file when this PR is merged.

Regards.
Claudiux

**Comment by rcalixte** [comment]
@claudiux Thank you for this applet and all your other contributions!

(And thank you for the French... mine is a little rusty. 😅)

**Comment by haggen88** [comment]
@rcalixte  if you wish you can add the Spanish translation :  [es.po.zip](https://github.com/linuxmint/cinnamon-spices-applets/files/10329773/es.po.zip)

**Comment by rcalixte** [comment]
Thank you again @haggen88! I've added your commit with you as the author! ¡Muchas gracias!

**Comment by claudiux** [comment]
Here the French translation.
[fr.po.zip](https://github.com/linuxmint/cinnamon-spices-applets/files/10331322/fr.po.zip)

**Comment by rcalixte** [comment]
> Here the French translation. [fr.po.zip](https://github.com/linuxmint/cinnamon-spices-applets/files/10331322/fr.po.zip)

Done and done! Merci beaucoup!

File: sound150@claudiux/files/sound150@claudiux/po/fr.po [hunk 1]
```diff
@@ -342,47 +344,47 @@ msgstr "Contrôle du volume maximum"
...
+msgstr "6%"
 
 #. 5.4->settings-schema.json->stepVolume->options
 #. 4.4->settings-schema.json->stepVolume->options
 #. 4.6->settings-schema.json->stepVolume->options
 #. 3.4->settings-schema.json->stepVolume->options
 #. 2.8->settings-schema.json->stepVolume->options
 msgid "5%"
-msgstr ""
+msgstr "±5%"
```
Comment (claudiux) [inline sound150@claudiux/files/sound150@claudiux/po/fr.po:355]: Please replace, in line 355, "±5%" by "5%". Sorry.

**Review by claudiux — CHANGES_REQUESTED** [review]

File: sound150@claudiux/files/sound150@claudiux/po/fr.po (hunk 1 above)
Comment (rcalixte) [reply sound150@claudiux/files/sound150@claudiux/po/fr.po]: No stress, it's an easy fix! Hopefully this is merged soon so it'll be easier to make future translation changes.

**Pull request merged**